lib/resources/content/javascriptservices/Views/Shared/_Layout.cshtml

```diff
@@ -4,8 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>@ViewData["Title"] - Aurelia</title>
-
-        <link rel="stylesheet" href="~/dist/vendor.css" asp-append-version="true">
```

Comment:
index.ejs states the styles will be added automatically here: https://github.com/artmasa/cli/blob/master/lib/resources/content/index-webpack.ejs